src/TLuaInterpreterUI.cpp

```diff
@@ -937,7 +937,16 @@ int TLuaInterpreter::getFont(lua_State* L)
     QString font;
     windowName = WINDOW_NAME(L, 1);
     auto console = CONSOLE(L, windowName);
-    font = console->mUpperPane->fontInfo().family();
+    Host& host = getHostFromLua(L);
+
+    if (console == host.mpConsole) {
+        font = host.getDisplayFont().family();
```

Comment:
`Host::getDisplayFont()` returns the value assigned to `QFont Host::mDisplayFont` - so it represents the value set in the past. However what gets used when that stored value is specified as the font is only accurately described by the `QFontInfo` class - see, e.g.: https://doc.qt.io/qt-6/qfontinfo.html#details !

Reply:
Done